message/messaging_service.cc

```diff
@@ -1093,6 +1093,9 @@ future<> messaging_service::unregister_schema_check() {
 future<utils::UUID> messaging_service::send_schema_check(msg_addr dst) {
     return send_message<utils::UUID>(this, netw::messaging_verb::SCHEMA_CHECK, dst);
 }
+future<utils::UUID> messaging_service::send_schema_check(msg_addr dst, abort_source& as) {
```

Comment:
How does it build up to the purpose of this patch set? And how does it work? How is it abortable?